src/Components/TicTacToe/TicTacToe.js

```diff
@@ -56,9 +56,8 @@ function checkWin(squares, size) {
   const lines = getLines(squares, size);
   for (let i = 0; i < lines.length; i++) {
     const line = lines[i];
-    if (line.every(e => e === line[0])) {
-      return line[0];
-    }
+    if (line.every(square => square === "X")) return "X";
+    if (line.every(square => square === "O")) return "O";
```

Comment:
Does this actually make a difference? Because it really feels like it shouldn't

Reply:
Yes I thought so too, but somehow it does.

Try filling the last row with player1 and then the middle row with player2 to reproduce the issue

Reply:
Ah, I guess it is because the previous version returns `null` if an entire line is empty